test/functional/cmdLineTests/gcCheck/build.xml

```diff
@@ -47,9 +47,11 @@
 		<echo>===executable:                   ${compiler.javac}</echo>
 		<echo>===debug:                        on</echo>
 		<echo>===destdir:                      ${DEST}</echo>
+		<copy todir="${src}/org/openj9/test/nogc">
+			<fileset dir="${allocator}" includes="Allocator.java" />
+		</copy>
```

Comment:
I think we should look for a solution that doesn't dirty the GIT working directory: there's the danger that the copy of `Allocator.java` will end up getting committed to the repository.

Reply:
Would add deleting this file in `clean` part of `build.xml` be a solution to solve this danger? Or deleting it immediately after compile? Thanks.

Reply:
It's not ideal using the working tree that way, and it's not necessary. The `<copy>` task can be removed and the `<javac>` task updated to this:
```
	<javac destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
		<src path="${src}" />
		<src path="${TEST_ROOT}/functional/Java8andUp/src" />
		<include name="org/openj9/test/nogc/Allocator.java" />
		<include name="org/openj9/test/nogc/Main.java" />
	</javac>
```
That way the right files are compiled without polluting the working tree unnecessarily.

Reply:
Thanks for the help. The commit has been updated and passed the test:
- hyc_jenkins: zlinux: 19834 (class can be veiw in artifacts)    zos: 19835
- Vmfarm: 17653